<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceQueryForm</class>
 <widget class="QWidget" name="DeviceQueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>672</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>舍    号:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="goatCheckBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>奶山羊绑定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>显示全部</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>状态:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>已绑定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>闲置</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>故障</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
